packages/@lwc/engine/src/framework/api.ts

```diff
@@ -609,10 +615,10 @@ export function ll(
     id: string,
     context?: (...args: any[]) => any
 ): EventListener {
+    const vmBeingRendered: VM | null = getVMBeingRendered();
```

Comment:
you should call this `vm` as it was before, because it is using in the next turn where `vmBeingRendered` is not longer set, it is confusing to call it like that.